### PR TITLE
actions cleanup

### PR DESCRIPTION
**Pull request by scottwn**
- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by scottwn** [comment]
Need to edit publish and publish_dev so that they are only building the service containers

**Comment by scottwn** [comment]
TODO: Fix the version tagging for the service images in ECR. Will be done as part of AgnostiqHQ/covalent-infra#23

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -57,20 +82,93 @@ jobs:
...
+        run: |
+          if [ ${{ github.event.inputs.prerelease }} = 'false' ] && \
+             [ -z ${{ github.event.inputs.stable_version }} ] ; then
+            echo "You can't create a stable release without specifying the stable version number."
+            exit 1
+          fi
+          VERSION="$(cat ./VERSION)"
+          cd dist
+          tar xzf cova-${VERSION}rc0.tar.gz
+          diff -x tests -r cova-${VERSION}rc0/covalent ../covalent
```
Comment (wjcunningham7) [inline .github/workflows/release.yml:151]: the name of the tarball will depend on whether it's a pre-release or stable release

**Review by wjcunningham7 — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (scottwn) [reply .github/workflows/release.yml:151]: oh right of course, thanks for catching that

**Review by wjcunningham7 — APPROVED** [review]

**Pull request merged**